src/cascadia/TerminalApp/TerminalSettings.cpp

```diff
@@ -104,6 +104,10 @@ namespace winrt::TerminalApp::implementation
             {
                 settings.StartingTabColor(static_cast<uint32_t>(til::color(newTerminalArgs.TabColor().Value())));
             }
+            if (newTerminalArgs.SuppressApplicationTitle())
+            {
+                settings.SuppressApplicationTitle(newTerminalArgs.SuppressApplicationTitle().Value());

```

Comment:
This is totally off-topic to this PR, but something I wanted to discuss out loud. Since we get the profile and generate a `TerminalSettings` from it, that means we get base layer (or profiles.defaults) applied properly. Maybe we should emphasize this in the docs? I feel like most people that use this setting want to enable it on every profile, but that's just a gut feeling.

Reply:
Actually I am not sure.. I usually use two types of tabs:
* General purpose ones, where I do change folders and stuff - I want the title to change
* Dedicated ones (e.g., git bash in terminal folder) that I don't  - I want the title to remain Terminal Dev.